tests/test_import_time.py

```diff
@@ -0,0 +1,48 @@
+import logging
+import subprocess
+import sys
+from pathlib import Path
+
+from datachain import C, DataChain
+
+logger = logging.getLogger(__name__)
+
+MAX_ATTEMPTS = 3
+MAX_IMPORT_TIME_MS = 700
+
+
+def _import_time_chain(test_session):
+    proc = subprocess.run(  # noqa: S603
+        [sys.executable, "-X", "importtime", "-c", "import datachain"],
+        stderr=subprocess.PIPE,
+        check=True,
+    )
+    out = proc.stderr.replace(b"import time:", b"").strip()
+    Path("import_time.csv").write_bytes(out)
+
+    dc = DataChain.from_csv("import_time.csv", session=test_session, delimiter="|")
+    dc = dc.save("import_time")
+    # columns are: cumulative_ms, self_ms, import
+    # TODO: use `mutate` instead of `map`
+    return (
+        dc.map(cumulative_ms=lambda cumulative: cumulative // 1000, output=int)
+        .map(self_ms=lambda self_us: self_us // 1000, output=int)
+        .map(**{"import": lambda imported_package: imported_package.lstrip()})
+    )
+
+
+def test_import_time(catalog, test_session):
```

Comment:
Amazing! Can we put some brief comment how would people approach fixing / profiling things in case it breaks. For a lot of folks it will be something new to learn and it would help if we have some steps / best practices here.

Reply:
Done. I have added a small comment on how to use the datachain to find the issue.
And also added a docstring with a how to analyze it yourself outside of the test.